source/includes/connection-snippets/x509-snippets/php-connection.php

```diff
@@ -0,0 +1,13 @@
+<!-- Begin x509 connection -->
```

Comment:
Same comments as I left on the SCRAM PHP example:

 * Don't use HTML comments
 * Start the source file with `<?php` and omit the final `?>
 * Use PHP comment syntax within the `<?php` tag

Also, I'll note that this file is using the PHP `'mongodb` extension (our low-level API) directly instead of the PHP library used in the SCRAM example. I think you should be consistent and use the library code, as it's what we have for most all PHP documentation examples.

So, construct a `MongoDB\Client` instead of a Manager object.